services/venice-controller/src/main/java/com/linkedin/venice/controller/StoreBackupVersionCleanupService.java

```diff
@@ -105,7 +105,7 @@ protected StoreBackupVersionCleanupService(
     });
     Optional<SSLFactory> sslFactory = admin.getSslFactory();
     this.httpAsyncClient = HttpAsyncClients.custom()
-        .setDefaultRequestConfig(RequestConfig.custom().setSocketTimeout(2000).build())
+        .setDefaultRequestConfig(RequestConfig.custom().setSocketTimeout(10000).build())
```

Comment:
Hmm, are you trying to set it to 10s? It seems very long timeout to me, could you give any context about this change? Like why it would take up to 10s..